ionic2/karma.conf.js

```diff
@@ -39,54 +30,51 @@ module.exports = function(config) {
     browserify: {
       debug: true,
       transform: [
-        ['browserify-istanbul', {
+        [require('browserify-istanbul'), {
           instrumenter: require('isparta'),
           ignore: ['**/*.spec.ts','**/*.d.ts'],
         }]
       ],
       plugin: [
-        ['tsify']
+        require('tsify')
       ]
     },
- 
- 
-    // test results reporter to use
-    // possible values: 'dots', 'progress'
-    // available reporters: https://npmjs.org/browse/keyword/karma-reporter
-    reporters: ['progress'],
- 
- 
+
+    customLaunchers: {
+      ChromeTravis: {
+        base: 'Chrome',
+        flags: ['--no-sandbox']
+      }
+    },
+
     // web server port
     port: 9876,
- 
- 
+
     // enable / disable colors in the output (reporters and logs)
     colors: true,
  
     proxies: {
-      '/build': '/base/www/build'
+      '/app': '/base/server/app',
+      '/app/api': 'http://pokedata.c4e3f8c7.svc.dockerapp.io:65014/api'
     },
  
     // level of logging
     // possible values: config.LOG_DISABLE || config.LOG_ERROR || config.LOG_WARN || config.LOG_INFO || config.LOG_DEBUG
     logLevel: config.LOG_INFO,
  
- 
     // enable / disable watching file and executing tests whenever any file changes
-    autoWatch: true,
- 
+    autoWatch: false,
  
     // start these browsers
     // available browser launchers: https://npmjs.org/browse/keyword/karma-launcher
-    browsers: ['Chrome'],
- 
- 
+    browsers: process.env.TRAVIS ? ['ChromeTravis'] : ['Chrome'],
```

Comment:
As far as I understand this refers to the `customLaunchers` property above. And it also seems to work fine:
https://travis-ci.org/PokemonGoers/Catch-em-all/builds/158177333